### PR TITLE
improve dev experience for launching eapp with saml auth

### PR DESCRIPTION
**Pull request by sojeri**
### Makefile & SAML docs improvements (c4081ff)

Adds a new operation to the Makefile
 - `run-saml` to spin up the whole cluster including SAML auth

Includes relevant documentation updates (see docs/saml.md)

### Docker volume improvements (799eafd)

Named some of the volumes we're using
- Left bind mounts alone
- Makes `docker volume ls` much easier to read
- Lets us specify how to persist the volumes

Specified local persistence
- Fixes issue we were seeing with wso2is state not persisting

## Checklist for Reviewer

- [ ] Review code changes
- [ ] Review changes for Database effects
- [ ] Verify change works in IE browser

More details about this can be found in [docs/review.md](docs/review.md)

### CODE REVIEW

**Comment by sojeri** [comment]
@macrael or @suzubara -- any thoughts on this PR? tagging you as I've talked to both of you about this already

**Comment by suzubara** [comment]
If I'm using the http://web:8080 config I get this error upon logging in 
image

File: docs/saml.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 # SAML
 
-To authenticate with SAML rather than the basic auth:
+The eQIP prototype uses wso2 identity server as its SAML provider. The exact distribution used is only currently available to folks within GSA, but for development purposes a fresh install of wso2is 5.8.0 will suffice.
```
Comment (ryanhofdotgov) [inline docs/saml.md]:
```suggestion
For historical reasons, [WSO2 Identity Server](https://wso2.com/identity-and-access-management/) is used to test SAML workflows with eApp. These instructions were written against WSO2 IAM 5.8.0, but generally any SAML 2.0 identity provider (IdP) supporting the [Web Browser SSO profile and HTTP redirect/HTTP POST bindings](https://en.wikipedia.org/wiki/SAML_2.0#SP_Redirect_Request;_IdP_POST_Response) for the service provider and identity provider respectively, should work.
```

File: docs/saml.md [hunk 2]
```diff
@@ -46,14 +54,39 @@ To authenticate with SAML rather than the basic auth:
...
+make run
+```
+
+Sometimes when cancelling the process, docker fails to shut down the containers. You can drop them with:
+
+```shell
+make stop-saml
+```
+
+### Login with PIV or CAC
```
Comment (ryanhofdotgov) [inline docs/saml.md]: Originally there was simultaneously support for simple authentication via a form as well as with a SAML provider. The button used the "Login with PIV/CAC" label, but really it meant "Login with SAML". Now the button should just say "Login". No PIVs or CACs are involved.

**Comment by ryanhofdotgov** [comment]
Asked someone from Soliel or ActioNet to pick up the review of this PR, validating the install instructions before approving.

**Pull request closed without merging**